src/styles.css

```diff
@@ -98,15 +98,45 @@ body {
 .mapboxgl-ctrl-group-removeaois {
   margin-top: 0px!important;
   border-radius: 0px 0px 4px 4px!important;
-  ;
   box-shadow: 0px 0px 0px 1px rgba(0, 0, 0, 0.1)!important;
 }
 
 .mapboxgl-ctrl-group-addgeobox {
   border-radius: 4px 4px 0px 0px!important;
-  ;
 }
 
+.maplibregl-ctrl-compass {
```

Comment:
Is there a reason why they were not put in the maplibre implementation?

Reply:
they act directly on the demo app only as it was the case for mapbox.
the styles are redone on arlas-wui